web/src/muted_topics_ui.js

```diff
@@ -1,6 +1,10 @@
+import $ from "jquery";
+
 import * as message_lists from "./message_lists";
+import * as overlays from "./overlays";
 import * as popover_menus from "./popover_menus";
 import * as recent_topics_ui from "./recent_topics_ui";
+import * as settings_user_topics from "./settings_user_topics";
 import * as stream_list from "./stream_list";
 import * as sub_store from "./sub_store";
```

Comment:
This file is still called `muted_topics_ui.js`. I think we should change that to `user_topics_ui.js` like we've done for the other related files.

Reply:
Makes sense. Addressed this in the 7th commit.